doc/source/release_notes.rst

```diff
@@ -1,6 +1,10 @@
 Release notes
 =============
 
+v0.3.4
+------
+- API: Make ``frame_shape`` match frames' ``shape`` in ``SpeStack`` (PR 241)
```

Comment:
Can you also say that the order has flipped?  The target audience for this text is someone who updated and now their code is broken so they may not be in the mood to dig through code / PRs :smiling_imp: .